staff/rares-deac/playground/portfolio/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+
+<head>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+
+        <div class="flex justify-center">
+
+            <h1 class="red-bg padding-5 radius-20">Rares Portfolio</h1>
+
+        </div>
+
+        <hr class="bold red-bg padding-5">
```

Comment:
format your code, because hr is not a container (at it seems to be showed as a container)